controllers/clusterversion_controller.go

```diff
@@ -86,7 +89,7 @@ func (c *ClusterVersionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			func(client client.Object) bool {
 				namespace := client.GetNamespace()
 				name := client.GetName()
-				return ((namespace == c.OperatorNamespace) && (name == operatorConfigMapName))
+				return ((namespace == c.OperatorNamespace) && (name == operatorConfigMapName) || (name == csiSidecarConfigMapName))
```

Comment:
I think the associativity changes here due to precedence of `&&` being higher than `||` and existing grouping does the same w/ or w/o it, so better be
```suggestion
				return namespace == c.OperatorNamespace && (name == operatorConfigMapName || name == csiSidecarConfigMapName)
```
btw, I hope you are looking for cm of operator/csisidecar from operatorns?